Content.Shared/Gibbing/Components/GibbableComponent.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Content.Shared.Gibbing.Systems;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Gibbing.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState, Access(typeof(GibbingSystem))]
+public sealed partial class GibbableComponent : Component
+{
+    /// <summary>
+    /// Giblet entity prototypes to randomly select from when spawning additional giblets
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public List<EntProtoId> GibPrototypes = new();
+
+    /// <summary>
+    /// Number of giblet entities to spawn in addition to entity contents
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public int GibCount;
+
+    /// <summary>
+    /// Sound to be played when this entity is gibbed, only played when playsound is true on the gibbing function
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public SoundSpecifier? GibSound = new SoundCollectionSpecifier("gib");
+
+    /// <summary>
+    /// Max distance giblets can be dropped from an entity when NOT using physics-based scattering
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float GibScatterRange = 0.3f;
+
+    /// <summary>
+    /// Audio parameters for when the gibbing sound is played
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public AudioParams GibAudioParams = AudioParams.Default.WithVariation(0.025f);
+}
```

Comment:
SoundSpecifier also has an AudioParams field you can use unless you want to standardise it across every collection.

Reply:
Might be worth having it as an override for the audio params on the sound specifier. Just in case two gibbable entities want to use the same sound but want to have different params.

Reply:
you can do just that every kind of sound collection / file / whatever supports params field